frontend/src/component/feature/FeatureToggleList/FeatureToggleListActions/FeatureToggleListActions.tsx

```diff
@@ -45,6 +47,11 @@ export const FeatureToggleListActions: FC<IFeatureToggleListActions> = ({
     const open = Boolean(anchorEl);
     const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
         setAnchorEl(event.currentTarget);
+        trackEvent('search-feature-buttons', {
+            props: {
+                clicked: 'group-actions',
```

Comment:
I'm wondering if we can come up with a better name than 'clicked' which is desktop vocabulary and doesn't apply to mobile tap. I'd prefer a business oriented name, not physical interface oriented name.

Reply:
Done